src/main/api/checkChangePasswordUsingLoginIdAndLoginIdTypes.json

```
@@ -0,0 +1,34 @@
+{
+  "uri": "/api/user/change-password",
+  "comments": [
+    "Check to see if the user must obtain a Trust Request Id in order to complete a change password request.",
+    "When a user has enabled Two-Factor authentication, before you are allowed to use the Change Password API to change",
+    "your password, you must obtain a Trust Request Id by completing a Two-Factor Step-Up authentication.",
+    "",
+    "An HTTP status code of 400 with a general error code of [TrustTokenRequired] indicates that a Trust Token is required to make a POST request to this API."
+  ],
+  "method": "get",
+  "methodName": "checkChangePasswordUsingLoginIdAndLoginIdTypes",
+  "successResponse": "Void",
+  "errorResponse": "Errors",
+  "params": [
+    {
+      "name": "loginId",
+      "comments": [
+        "The loginId of the User that you intend to change the password for."
+      ],
+      "type": "urlParameter",
+      "parameterName": "loginId",
+      "javaType": "String"
+    },
+    {
+      "name": "loginIdTypes",
+      "comments": [
+        "the identity types that FusionAuth will compare the loginId to."
```

There are a couple other APIs that have the `t` for this parameter. Can we fix those too?
```suggestion
        "The identity types that FusionAuth will compare the loginId to."
```

---

Pushed in 0ddacb567c00c0076d2a5ebae256b15921f3d713 - everything will need re-approval as a result (app especially)